sql/moz-fx-data-shared-prod/fenix_derived/new_profile_activation_clients_v1/backfill.yaml

```diff
@@ -0,0 +1,8 @@
+2025-02-01:
+  start_date: 2021-01-01
+  end_date: 2025-02-01
+  reason: Initial backfill for the table
+  watchers:
+  - kik@mozilla.com
+  status: Initiate
+  shredder_mitigation: false
```

Comment:
Need to set add override_retention to true?
Also, I am not very sure if this will work for future dates.